app/lib/configuration.js

```diff
@@ -20,6 +20,7 @@ class Configuration {
     this.theme = ''
     this.hotkey = ''
     this.debug = false
+    this.height = 700
   }
 
   load () {
```

Comment:
Further down in this file we load and assign the `data` you need to setup `this.height = data.height || 400` or something similar.